kubernetes/rucio/README.md

```diff
@@ -0,0 +1,146 @@
+Adapted from rucio instructions here: https://github.com/rucio/helm-charts/tree/master/rucio-server
+
+
+# First time setup
+
+## Setup a new cluster in the CMSRucio project:
+
+
+    openstack coe cluster delete --os-project-name CMSRucio  cmsruciotest
+    openstack coe cluster create cmsruciotest --keypair lxplus  --os-project-name CMSRucio   --cluster-template kubernetes-preview --node-count 4
+    openstack coe cluster list --os-project-name CMSRucio # Monitor creation status
+
+
+### If setting up a new/changed cluster:
+
+
+    rm key.pem cert.pem ca.pem config
+    openstack coe cluster config  --os-project-name CMSRucio  cmsruciotest
```

Comment:
Please explicitly specify usage of `CMSRucio` project name since it is project-name namespace with quota on OpenStack. If user don't have it the instructions are confusing or at least users should not specify it. A simple comment here would be sufficient.